src/components/UserList/index.tsx

```diff
@@ -87,6 +87,9 @@ const UserList: React.FC = () => {
       pageIndex * currentPageSize
     }&sort=${currentSort}`
   );
+  const { data: emailSettings } = useSWR(
+    '/api/v1/settings/notifications/email'
+  );
```

Comment:
Users need the manage settings permission to hit this endpoint but they only need `manage users` to create a user.

Reply:
Ah. How can we verify that email notifications are enabled, then? 🤔  This would also be an issue for my other PR.

Reply:
I'm exposing the "email enabled" state via the user notification settings endpoint in #1172, so marking this PR as a draft for now.